packages/fxa-payments-server/src/components/NewUserEmailForm/index.test.tsx

```diff
@@ -163,23 +163,6 @@ describe('NewUserEmailForm test', () => {
     });
   });
 
-  it('shows error when an account already exists', async () => {
```

Comment:
Was having some race condition issues in this test because of the new debounce function. The below test actually does a better job of testing this functionality, so I removed this test.

Reply:
@meandavejustice can you link to the test you are referring to?  I don't see a new test in the PR.

Reply:
https://github.com/mozilla/fxa/pull/10137/files/11896dfbe5d68910d61605d1e84f5d3516179605#diff-e46a5e7e3b8430f12dcebce9d576a447060f43f87b6cdce9fdbc4007a26dbe0aR166

Reply:
That link didn't work.

Reply:
https://github.com/mozilla/fxa/pull/10137/files#diff-e46a5e7e3b8430f12dcebce9d576a447060f43f87b6cdce9fdbc4007a26dbe0aR166

It is the test directly under this one in the file